docs/provider-specific.md

```diff
@@ -20,11 +20,19 @@ The file is then downloaded using the API to find the best server according to t
 * selecting server (limit answer to 1): [ANSWER]= https://api.nordvpn.com/v1/servers/recommendations?filters[country_id]=2&filters[servers_technologies][identifier]=openvpn_tcp&filters[servers_group][identifier]=legacy_group_category&limit=1
 * download selected server's config: https://downloads.nordcdn.com/configs/files/ovpn_[NORDVPN_PROTOCOL]/servers/[ANSWER.0.HOSTNAME][] => https://downloads.nordcdn.com/configs/files/ovpn_tcp/servers/al9.nordvpn.com.tcp.ovpn
 
-A possible evolution would be to check server's load to select the most available one.
+One optional ENV var NORDVVPN_TESTS can take value from 1 to 3. Expected generic results are written to logs.
```

Comment:
Also
`NORDVVPN_TESTS` should be
`NORDVPN_TESTS`

Reply:
Thanks for noticing typo. I changed doc and fixed download error. I'll submit another PR.